src-docs/src/views/datagrid/schema.js

```diff
@@ -180,6 +180,7 @@ const DataGridSchema = () => {
       schemaDetectors={[
         {
           type: 'favoriteFranchise',
+          textTransform: 'lowercase',
```

Comment:
```suggestion
          textTransform: 'capitalize',
```

Just noting that the property is there without altering appearance.